tools/openblas_support.py

```diff
@@ -0,0 +1,42 @@
+import os
+import textwrap
+
+def make_init(dirname):
+    '''
+    Create a _distributor_init.py file for OpenBlas
+    '''
+    with open(os.path.join(dirname, '_distributor_init.py'), 'wt') as fid:
```

Comment:
Why not make this the file in tools and copy it into the root?

Reply:
That is, name this `_distributor_init.py`.

Reply:
The command might be
```
Copy-Item -Path tools\_distributor_init.py -Destination .
```

Reply:
My goal is to expand this file to include the download-and-expand parts of getting OpenBlas from rackspace, so we don't have separate stanzas in each CI system. Right now each OS on each CI system handles this differently, there is no single place that specifies the OpenBLAS version and root fo the rackspace directory. So it seems natural to keep all the OpenBLAS handling in one file